plugins/catalog-react/src/components/EntityRefLink/EntityRefLink.tsx

```diff
@@ -78,16 +202,30 @@ export const EntityRefLink = forwardRef<any, EntityRefLinkProps>(
     );
 
     const link = (
-      <Link {...linkProps} ref={ref} to={entityRoute(routeParams)}>
+      <Link
+        {...bindHover(popupState)}
+        {...linkProps}
+        ref={ref}
+        to={entityRoute(routeParams)}
+      >
         {children}
         {!children && (title ?? formattedEntityRefTitle)}
       </Link>
     );
 
-    return title ? (
-      <Tooltip title={formattedEntityRefTitle}>{link}</Tooltip>
-    ) : (
-      link
+    return (
+      <>
+        {title ? (
+          <Tooltip title={formattedEntityRefTitle}>{link}</Tooltip>
```

Comment:
Does this mean it'll have a tooltip AND a popover? Does that make sense visually?

Reply:
good point.